appveyor.yml

```diff
@@ -36,24 +23,25 @@ install:
       https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
       Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
         throw "There are newer queued builds for this pull request, failing early." }
-  - "git clone git://github.com/astropy/ci-helpers.git"
-  - "powershell ci-helpers/appveyor/install-miniconda.ps1"
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "conda create -y -n test python=%PYTHON_VERSION%"
```

Comment:
It's not a review, it's a question. I hope you don't mind me asking here.

Instead of creating a new environment, would it have been ok to define the variable "PYTHON" in the matrix instead as PYTHON: "C:\\Miniconda27-x64" and `PYTHON: "C:\\Miniconda36-x64"`, respectively, for Python 2.7 and 3.6 and test directly in the base environment?

I think that may reduce the installation time, but I don't know if there is any downside to that.